### DISCUSSION TITLE
Duda complejidad insertar en arboles

### DISCUSSION THREAD

**Discussion by RodolfoKompatzki** [IIC2133-PUC/2024-1#103] [Tarea 1] [answered]
Hola, al construir los arboles de la parte 2, tengo duda si cuenta la complejidad de INSERTAR las cabezas. Considerando que la complejidad de búsqueda se cumple, mi duda es particularmente por la operación de insertarlas en el árbol.

**Comment by sebacarrascop** [comment, marked as answer]
hola!, esa complejidad es algo aparte, debiese ser nlog(n) o n^2 en el peor caso (por las n cabezas), aunque no se pide explícitamente.
